web/css/main.css.map

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
The reason I checked it in is so that it works out of the box on checkout/install. But I'm not adverse to leaving it out (for the reasons you gave)